README.md

```diff
@@ -426,7 +426,7 @@ A property attribute MUST be one of these:
             <code>#</code> Count or Amount
         </td>
         <td>Yes</td>
-        <td>Yes.<br /> If the property is unit-less, e.g. a discrete state, set this to <code>none</code></td>
+        <td>No ("")</td>
```

Comment:
A MQTT message with a zero-length content will delete the associated retained message, which is why we chose `none`

Reply:
Understood.
However, the default is not what is published onto the MQTT topic, the default is the value that will be used by a controller IF the topic is NOT published.

Reply:
Oh, got you. Did not notice the `(default)`